### ISSUE TITLE
Global Tables working like a local table

### ISSUE THREAD

**Issue by DhruvaPatil98** [robinhood/faust#451]
## Checklist

- [x] I have included information about relevant versions
- [x] I have verified that the issue persists when using the `master` branch of Faust.

## Steps to reproduce

Define a global table and assign values to some keys

```pytb
table = app.GlobalTable(
    'event_table',
    partitions=2,
)
```
## Expected behavior

All workers must be able to access those keys

## Actual behavior

The workers only have access to their own keys and values. Issue remains when using 1.8, 1.8.1 and master branch

## Full traceback

This is the data I get when I print out all the keys and values in the global table in each worker (key and value given are same)
```pytb
Worker 1:
[2019-10-19 12:52:19,829] [54] [WARNING] d: d 
[2019-10-19 12:52:19,829] [54] [WARNING] f: f 
[2019-10-19 12:52:19,830] [54] [WARNING] h: h 
Worker 2:
[2019-10-19 12:52:27,370] [55] [WARNING] b: b 
[2019-10-19 12:52:27,370] [55] [WARNING] c: c 
[2019-10-19 12:52:27,370] [55] [WARNING] e: e 
[2019-10-19 12:52:27,371] [55] [WARNING] i: i
```
I also keep getting this message (issue referenced in #450 )
```pytb
[2019-10-19 13:00:02,732] [54] [WARNING] [^--Consumer]: Possible livelock: COMMIT OFFSET NOT ADVANCING FOR TopicPartition(topic='app_main-event_table-changelog', partition=0) 
```
## Further Information regarding setup

I am using Faust inside docker. Running multiple workers on different services.
Here is the docker-compose file
```pytb
version: '2'
services:

  zookeeper:
    image: wurstmeister/zookeeper
    ports:
      - "2181:2181"

  kafka:
    image: wurstmeister/kafka
    ports:
      - "9092:9092"
      - "9094:9094"
    environment:
      KAFKA_ADVERTISED_LISTENERS: INSIDE://:9092,OUTSIDE://kafka:9094
      KAFKA_LISTENERS: INSIDE://:9092,OUTSIDE://:9094
      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: INSIDE:PLAINTEXT,OUTSIDE:PLAINTEXT
      KAFKA_INTER_BROKER_LISTENER_NAME: INSIDE
      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
      KAFKA_MESSAGE_MAX_BYTES: 999999999
      KAFKA_LOG_DIRS: /kafka/kafka-logs-1
    volumes:
      - kafka_data:/kafka

    depends_on:
      - zookeeper

  redis:
    image: 'bitnami/redis:5.0'
    environment:
      # ALLOW_EMPTY_PASSWORD is recommended only for development.
      - ALLOW_EMPTY_PASSWORD=yes
      - REDIS_DISABLE_COMMANDS=FLUSHDB,FLUSHALL
    ports:
      - '6379:6379'
    volumes:
      - 'redis_data:/bitnami/redis/data'

  app1:
    build: .
    volumes:
      - ./server:/application
      - app-python-site-packages1:/usr/local/lib/python3.6/site-packages
      - app-usr-local-bin1:/usr/local/bin
      - worker-data1:/data/
    # command: "faust --datadir=/data/work_6067 -A app worker -l info --web-port 6067"
    command: tail -f /dev/null
    ports:
      - "6067:6067"
    depends_on:
      - kafka

  app2:
    build: .
    volumes:
      - ./server:/application
      - app-python-site-packages2:/usr/local/lib/python3.6/site-packages
      - app-usr-local-bin2:/usr/local/bin
      - worker-data2:/data/
    # command: "faust --datadir=/data/work_6068 -A app worker -l info --web-port 6068"
    command: tail -f /dev/null
    ports:
      - "6068:6068"
    depends_on:
      - kafka


volumes:
  kafka_data:
    driver: local

  redis_data:
    driver: local

  app-python-site-packages1:
    driver: local

  app-usr-local-bin1:
    driver: local

  app-python-site-packages2:
    driver: local

  app-usr-local-bin2:
    driver: local

  worker-data1:
    driver: local

  worker-data2:
    driver: local

```
From [archit0/faust-docker-compose](https://github.com/archit0/faust-docker-compose)
# Versions

* Python version - 3.7.4
* Faust version - 1.8.1 (doesn't seem to work in 1.8 either)
* Operating system
* Kafka version - 1.4.6
* RocksDB version (if applicable) - 0.7.0

**Comment by DhruvaPatil98** [comment]
This issue still persists. I have fixed it with Pull Request #462

**Comment by archit0** [comment]
@ask can you look into this p.r https://github.com/robinhood/faust/pull/462
